### ISSUE TITLE
Selectable tile markup does not show checkmark

### ISSUE THREAD

**Issue by emyarod** [carbon-design-system/carbon#1741]
Related: https://github.com/IBM/carbon-components-react/issues/1819

The legacy selectable tile markup currently contains a11y violations (https://github.com/IBM/carbon-components-react/issues/1114). This was addressed in the React repo via https://github.com/IBM/carbon-components-react/pull/1774, but the CSS no longer targets the new accessible markup.
